packages/project-access/package.json

```diff
@@ -34,12 +34,15 @@
     "dependencies": {
         "@sap-ux/i18n": "workspace:*",
         "@sap-ux/ui5-config": "workspace:*",
+        "fast-xml-parser": "4.2.7",
         "findit2": "2.2.3",
         "mem-fs": "2.1.0",
-        "mem-fs-editor": "9.4.0"
+        "mem-fs-editor": "9.4.0",
```

Comment:
this is an older version? Should we bump this https://www.npmjs.com/search?q=mem-fs-editor

Reply:
I guess this is in green justt because the `,` was added. I didn't update the version